### ISSUE TITLE
Network-manager indicator missing from guest session

### ISSUE THREAD

**Issue by asokero** [opinsys/puavo#58]
While logged in as a Guest user, the network-manager indicator is missing from the gnome-panel.

**Comment by juhaerk** [comment]
Apparently guest-users do not have equivalent rights to normal users, and we do not know why (we should know).

Trying to run nm-applet as a guest user gives the following warning:

```
** (nm-applet:28264): WARNING **: Could not initialize NMClient /org/freedesktop/NetworkManager: Rejected send message, 3 matched rules; type="method_call", sender=":1.638" (uid=128 pid=28264 comm="nm-applet ") interface="org.freedesktop.DBus.Properties" member="GetAll" error name="(unset)" requested_reply="0" destination="org.freedesktop.NetworkManager" (uid=0 pid=1259 comm="NetworkManager ")

...

(nm-applet:28264): nm-applet-WARNING **: Failed to register as an agent: (32) Session not found
```

The first error message ("Rejected send message") can be created by running the following command as a guest user:

```
dbus-send --system --print-reply --dest=org.freedesktop.NetworkManager \
          /org/freedesktop/NetworkManager \
          org.freedesktop.DBus.Properties.GetAll
```

Policy files do not reveal any apparent reason for this (or at least I could not figure a reason for this).

**Comment by asokero** [comment]
It seems that after device boot the first guest-user is able to see and change the network-manager settings. After logging out and back in as a guest the network-manager indicator is not visible anymore.

**Comment by asokero** [comment]
On webkiosk desktop this is actually a needed feature so that customers would not accidentally make the device unusable.

**Comment by juhaerk** [comment]
Fixed as a side effect of a fix to this ticket: #102.
